### PR TITLE
Add more verbose logging on missing files

### PR DESCRIPTION
**Pull request by akram**
Add more verbose logging on missing files

### CODE REVIEW

File: vendor/github.com/syndtr/goleveldb/leveldb/errors/errors.go [hunk 1]
```diff
@@ -64,7 +64,10 @@ type ErrMissingFiles struct {
 	Fds []storage.FileDesc
 }
 
-func (e *ErrMissingFiles) Error() string { return "file missing" }
+func (e *ErrMissingFiles) Error() string { 
+	msg := fmt.Sprintf("file missing : %s ", e.Fds )
```
Comment (grobie) [inline vendor/github.com/syndtr/goleveldb/leveldb/errors/errors.go]: Let's remove the spaces before the colon and before the closing round bracket.

File: vendor/github.com/syndtr/goleveldb/leveldb/errors/errors.go [hunk 2]
```diff
@@ -64,7 +64,10 @@ type ErrMissingFiles struct {
 	Fds []storage.FileDesc
 }
 
-func (e *ErrMissingFiles) Error() string { return "file missing" }
+func (e *ErrMissingFiles) Error() string {
+	msg := fmt.Sprintf("file missing : %s ", e.Fds)
+	return msg
+}
```
Comment (fabxc) [inline vendor/github.com/syndtr/goleveldb/leveldb/errors/errors.go]: Editing a vendored file in this way is unfortunately not okay. We've no proper means to ensure that change sticks around. You'd have to upstream it first.

**Comment by beorn7** [comment]
@fabxc is right. We import LevelDB as a library and treat it essentially as a black box.

The good news is that there are tools out there to debug a corrupt LevelDB in cold state. (Shut Prometheus down and then use those tools on the files on disk.)

**Comment by beorn7** [comment]
@akram I'll close this PR. If you find a way to solve the problem within the Prometheus code base, feel free to re-open / re-create.

**Pull request closed without merging**